### DISCUSSION TITLE
Documentation - UI components and extensions

### DISCUSSION THREAD

**Discussion by hergerr** [directus/directus#16573] [Feature Requests]
Hi. I am working with Directus and I wish there were more examples on eg. building custom module extension. For the most part, laborious analyze of the source code is needed. 

To name an example, it would be great, if there was more documentation on data validation in v-form. It would be awesome if type like ValidationError would be better documented and some example would be present in docs.

Thanks!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
